lib/binance/supervisor.ex

```diff
@@ -1,23 +0,0 @@
-defmodule Binance.Supervisor do
-  @moduledoc """
-  Supervisor for cache and other required persistence layers
-  """
-
-  use Supervisor
-
-  def start_link(opts \\ []) do
-    Supervisor.start_link(__MODULE__, :ok, opts)
-  end
-
-  def init(:ok) do
-    children = [
-      Binance.SymbolCache
-    ]
-
-    Supervisor.init(children, strategy: :one_for_one)
-  end
-
-  def start(_type, _args) do
-    start_link()
-  end
-end
```

Comment:
Esse arquivo acho que é necessário pra rodar o app. Ele que deve ser responsável pela incialização de alguns processos. Rodei com essa branch desatualizada e tudo funcionou :partying_face:. Podemos fazer alguns arquivos de quais outros arquivos são necessários.

Reply:
Só faltou remover a inicialização desse arquivo do `mix.exs` 😅.

Testei aqui com as duas branches mais atualizadas e parece estar tudo funcionando 🎉

Reply:
Só pra explicar o motivo de ter removido esse arquivo:

Esse módulo só servia como cache pra estrutura de `%Binance.TradePair{}` que existia, que possibilitava mandar algo como `%{from: "BTC", to: "USDT"}` ao invés de `"BTCUSDT"`. Com a forma de struct, ele fazia um request pra binance pra ver se o par existia, e em qual ordem (`BTCUSDT` vs `USDTBTC`). Esse módulo só servia como cache pra não ter que mandar a request de novo caso usasse um tradepair repetido